VideoLocker/src/main/java/org/edx/mobile/view/VideoListActivity.java

```diff
@@ -195,10 +195,7 @@ public void run() {
             }
 
             String filepath = null;
-            // check if file available on local
-            if (video.isVideoForWebOnly) {
```

Comment:
@BenjiLee I believe I took that out because it was a warning. Android reported that was always false and that the code was never executed. It's certainly not a part of the core logic.

If it makes you more comfortable I'll put it back in. In fact, I'dd just do that.